File: CryBot.Web/ClientApp/app/models/api/ICryptoOrder.ts
import { CryptoOrderType } from "../CryptoOrderType";

export interface ICryptoOrder {
    market: string;
    orderType: CryptoOrderType;
    price: number;
    quantity: number;
    pricePerUnit: number;
    commissionPaid: number;
    canceled: boolean;
    uuid: string;
    opened: Date;
    limit: number;
    quantityRemaining: number;
    closed: Date;
    isClosed: boolean;
}

File: CryBot.Core/Models/ICryptoOrder.cs
using System;

namespace CryBot.Core.Models
{
    public interface ICryptoOrder
    {
        string Market { get; set; }
        CryptoOrderType OrderType { get; set; }
        decimal Price { get; set; }
        decimal Quantity { get; set; }
        decimal PricePerUnit { get; set; }
        decimal CommissionPaid { get; set; }
        bool Canceled { get; set; }
        string Uuid { get; set; }
        DateTime Opened { get; set; }
        decimal Limit { get; set; }
        decimal QuantityRemaining { get; set; }
        DateTime Closed { get; set; }
        bool IsClosed { get; set; }
    }
}